src/pages/status.js

```diff
@@ -157,7 +205,7 @@ export default function StatusPage() {
                 </tr>
               </thead>
               <tbody>
-                <StatusRows components={components} filter={filter} />
+                <StatusRows components={components} filter={selectedField.filter} />
```

Comment:
Non-blocking nitpick: if we have a property called `filter`, it could look a little strance since `.filter()` is an array method.

Reply:
Good point. I'll change it to `type`